### PR TITLE
release(protocol): release to update core to include solidity 0.8

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

Release latest code to update core to include 0.8 solidity version.

### CODE REVIEW

File: packages/affiliates/package.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
   "license": "MIT",
   "dependencies": {
     "@google-cloud/bigquery": "^5.3.0",
-    "@uma/common": "^2.1.0",
-    "@uma/core": "^2.2.0",
-    "@uma/financial-templates-lib": "^2.2.0",
+    "@uma/common": "^2.2.0",
+    "@uma/core": "^2.3.0",
+    "@uma/financial-templates-lib": "^2.3.0",
+    "@uniswap/v3-core": "1.0.0-rc.2",
```
Comment (nicholaspai) [inline packages/affiliates/package.json:11]: Why adding these now? + `prompt`

**Review by nicholaspai — APPROVED** [review]
LGTM , had 1 question about a couple packages added to `affiliates`

File: packages/affiliates/package.json (hunk 1 above)
Comment (chrismaree) [reply packages/affiliates/package.json:11]: not sure why this changed bth. @mrice32 any idea?

Comment (chrismaree) [reply packages/affiliates/package.json:11]: o ye. that's exactly what it did.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**